source/companies.json

```diff
@@ -76,6 +76,11 @@
             "websiteUrl": "http://www.google.com",
             "description": "Google LLC is an American multinational technology company that specializes in Internet-related services and products, which include online advertising technologies, search engine, cloud computing, software, and hardware."
         },
+        "isolarcloud": {
```

Comment:
Maybe `Sungrow Power Supply Company Limited` could be added to the company and as a product `isolarcloud` to the trackers